.env

```diff
@@ -1,4 +1,5 @@
 PUBLIC_URL=https://arquisoft.github.io/viade_es2b/
 REACT_APP_VERSION=$npm_package_version
 REACT_APP_NAME=$npm_package_name
-REACT_APP_COMPANY_NAME=University of Oviedo
\ No newline at end of file
+REACT_APP_COMPANY_NAME=University of Oviedo
+PORT=4000
```

Comment:
Why do you change that?

Reply:
Local deployment does not work on default port 3000 on my pc, that's the reason.